Storage/SQL/BrowserDB.swift

```diff
@@ -56,7 +56,7 @@ public class BrowserDB {
         let file = ((try! files.getAndEnsureDirectory()) as NSString).stringByAppendingPathComponent(filename)
         self.db = SwiftData(filename: file, key: secretKey, prevKey: nil)
 
-        if AppConstants.BuildChannel == .Developer && secretKey != nil {
+        if AppConstants.BuildChannel == .Fennec && secretKey != nil {
```

Comment:
IMO, `Developer`/`Debug`/`Local`/etc. would be more intuitive than `Fennec`. If I see `Fennec`, I don't immediately know what that means without looking at the other possible flags and using the process of elimination.